tests/queries/0_stateless/02346_text_index_bug87887.sql

```diff
@@ -0,0 +1,12 @@
+-- Test for Bug 87887
+
+SET allow_experimental_full_text_index = 1;
```

Comment:
don't you have to set these settings to enable the direct read?

```sql
SET use_skip_indexes_on_data_read = 1;
SET use_skip_indexes = 1;
SET query_plan_direct_read_from_text_index = 1;
```

Reply:
Yes, this ^^ will make sense.

Reply:
Solved via [85d0133](https://github.com/ClickHouse/ClickHouse/commit/85d01339c74760c44351fe0e1c0ca044a96d047b)